grow/pods/catalogs.py

```diff
@@ -118,13 +122,18 @@ def init(self, template_path, include_header=False):
         self.fuzzy = False
         self.save(include_header=include_header)
 
-    def update_using_catalog(self, catalog_to_merge, use_fuzzy_matching=False):
+    def update_using_catalog(self, catalog_to_merge, use_fuzzy_matching=False,
+                             include_obsolete=False):
         super(Catalog, self).update(
             catalog_to_merge, no_fuzzy_matching=(not use_fuzzy_matching))
+        # Don't use gettext's obsolete functionality as it polutes files: merge
```

Comment:
@jeremydw Meant to ask — do you agree with this? Looks like the current implementation doesn't use gettext's functionality for obsolete strings, so I avoided it too: if you `include_obsolete` it'll just include obsolete terms as if they weren't obsolete. Seemed safest, in case obsolete syntax isn't widely supported by translation tools (dunno if this is the case, just a guess it might not be!)